lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -523,6 +529,36 @@ SwiftLanguageRuntimeImpl::GetSwiftMetadataCache() {
   return &m_swift_metadata_cache;
 }
 
+std::vector<std::string>
+SwiftLanguageRuntimeImpl::GetConformances(llvm::StringRef mangled_name) {
+  if (m_conformances.empty()) {
+    using namespace swift::Demangle;
+    Demangler dem;
+
+    ThreadSafeReflectionContext reflection_ctx = GetReflectionContext();
+    if (!reflection_ctx)
+      return {};
+
+    Progress progress("Downloading Swift conformances");
```

Comment:
Maybe "parsing" instead of "downloading"?

Reply:
+1, or "Loading"